<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ParamsButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Params File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TorqueButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Input Torque</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OutputButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert Output File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CalculateButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="PauseBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>76</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OpenButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Output File</string>
    </property>
   </widget>
   <widget class="QLabel" name="PauseLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Press to Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="ParamsLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No file added</string>
    </property>
   </widget>
   <widget class="QLabel" name="InsertOutLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>220</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No file added</string>
    </property>
   </widget>
   <widget class="QLabel" name="CalculateLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Press to Calculate</string>
    </property>
   </widget>
   <widget class="QLabel" name="OpenOutLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Press to open the output data</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No file added</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>340</y>
      <width>151</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Output File</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>340</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Press to delete</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>608</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
